inspector/src/components/actionTabs/tabs/propertyGrids/animations/animationCurveEditorComponent.tsx

```diff
@@ -296,24 +296,50 @@ export class AnimationCurveEditorComponent extends React.Component<
     }
   }
 
+  getKeyframeValueFromAnimation(id: string) {
+    const animation = this.state.selected as Animation;
+    const index = parseInt(id.split('_')[3]);

```

Comment:
Those constructs look error prone ?

Reply:
I have an SVG curve id created based on the coordinate, the key index, and the name. This is to have a way to match the Keyframe in the animation and the SVG representation. Let me think of another way to sync those...

Reply:
If not possible it is fine if you create it as well and this is not a client input. but in this case, maybe adding a helper encode/decode would make it more obvious and less subject to maintenance issues.

Reply:
Ah. Great idea! I'm using this in a couple of places. So, yes, this is a benefit. I'll create that method/function